wpilibc/athena/include/Compressor.h

```diff
@@ -18,7 +18,17 @@
 namespace frc {
 
 /**
- * PCM compressor
+ * Class for operating a compressor connected to a %PCM (Pneumatic Control
+ * Module). The %PCM will qutomatically run in closed loop mode by default
+ * whenever a Solenoid object is created. For most cases, a Compressor object
```

Comment:
Doxygen does it automagically

![image](https://user-images.githubusercontent.com/4885091/29483718-3c8297de-8462-11e7-8415-05754aec117b.png)